pegasus/sites.v3/code.org/public/advocacy/state-facts/AK.pdf.fetch

```diff
@@ -1 +1 @@
-https://cdo-fetch.s3.amazonaws.com/6e4725247daef51796914d5e17728ae0-AK.pdf
\ No newline at end of file
+https://cdo-fetch.s3.amazonaws.com/d0b7963fd016c1b75c1e33ab8ee94f19-AK.pdf
```

Comment:
I thought we normally pick up new PDF files via a staging scoop, though this might be okay too...